backend/src/main/resources/schema-mysql.sql

```diff
@@ -0,0 +1,113 @@
+USE `MovingSystem-DB`;
+
+CREATE TABLE IF NOT EXISTS Client (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    clientId VARCHAR(36) UNIQUE NOT NULL,
+    firstName VARCHAR(255),
+    lastName VARCHAR(255),
+    email VARCHAR(255) UNIQUE,
+    phoneNumber VARCHAR(15)
+    );
+
+CREATE TABLE IF NOT EXISTS Truck (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    vin VARCHAR(17) UNIQUE NOT NULL,
+    capacity DOUBLE
+    );
+
+CREATE TABLE IF NOT EXISTS Shipment (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    shipmentId VARCHAR(36) UNIQUE NOT NULL,
+    shipmentStatus VARCHAR(255),
+    shipmentDepartureAddress VARCHAR(255),
+    shipmentArrivalAddress VARCHAR(255),
+    vin VARCHAR(17),
+    clientId VARCHAR(36),
+    FOREIGN KEY (vin) REFERENCES Truck(vin),
+    FOREIGN KEY (clientId) REFERENCES Client(clientId)
+    );
+
+CREATE TABLE IF NOT EXISTS Inventory (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    inventoryId VARCHAR(36) UNIQUE NOT NULL,
+    description VARCHAR(255),
+    shipmentId VARCHAR(36),
+    FOREIGN KEY (shipmentId) REFERENCES Shipment(shipmentId)
+    );
+
+CREATE TABLE IF NOT EXISTS Item (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    itemId VARCHAR(36) UNIQUE NOT NULL,
+    itemName VARCHAR(255) NOT NULL,
+    type ENUM('ITEM', 'BOX') NOT NULL,
+    price DECIMAL(10, 2),
+    description VARCHAR(255),
+    weight DOUBLE NULL,
```

Comment:
Similar to the capacity attribute in the truck table, later on since the weight could be in kg or lbs, this could be changed to a string
```suggestion
    weight VARCHAR(5) NULL,
```